src/watch/hearRingiOSWatchOS/hearRingiOSWatchOS.xcodeproj/project.pbxproj

```diff
@@ -885,7 +885,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
```

Comment:
this is nitpicking but like if we are pushing to main I think it may be best to stay with our convention of increasing the number of the bundle identifier by 1? It doesn't really matter though, idk